content/en/docs/languages/csharp/_index.md

```diff
@@ -1,21 +1,32 @@
 ---
-title: C#
-description: The original core-library implementation of gRPC for C#
+title: C# / .NET
 api_path: grpc/LANG/api/Grpc.Core
-prog_lang_home: true
-src_repo: https://github.com/grpc/grpc
+src_repo: https://github.com/grpc/grpc-dotnet
 content:
-  - learn_more:
-    - "[Additional docs]($src_repo_url/tree/master/doc/csharp)"
-    - "[Examples]($src_repo_url/tree/master/examples/csharp)"
   - reference:
     - "[API](api/)"
   - other:
-    - "[gRPC for .NET](dotnet/)"
-    - "[grpc repo]($src_repo_url)"
     - "[Daily builds](daily-builds)"
-cascade:
-  - show_banner: true
 ---
 
-{{% docs/prog-lang-home-content %}}
+*This page used to contain the documentation for the original C# implementation
+of gRPC based on the native gRPC Core library (i.e. `Grpc.Core` nuget package).
+The implementation is currently in maintenance mode and its source code has
+been [moved][move-details]. We plan to deprecate
+the implementation in the future (see [blogpost][]) and we recommend that
+all users use the [grpc-dotnet][] implementation instead.*
```

Comment:
This would probably make sense inside an `alert`? WDTY?

Reply:
Actually, "alert" will break the links, so I'm not going to use it.
![image](https://user-images.githubusercontent.com/9939684/199014046-bcfc667f-e3eb-44fc-b4d2-76627de62967.png)